tests/ui/unpretty/deprecated-attr.stdout

```diff
@@ -5,17 +5,21 @@ extern crate std;
 //@ compile-flags: -Zunpretty=hir
 //@ check-pass
 
-#[deprecated]
+#[attr = Deprecation {deprecation: Deprecation {since: Unspecified}}]
```

Comment:
some of the spacing is a *bit* tighter than i'd like, but the unpretty machinery is annoying lol